doc/manual/src/glossary.md

```diff
@@ -59,18 +59,17 @@
 
 - [store]{#gloss-store}
 
-  A collection of store objects, with operations to manipulate that collection.
-  See [Nix store](./store/index.md) for details.
+  A collection of [store objects][store object], with operations to manipulate that collection.
+  See [Nix Store](./store/index.md) for details.
 
-  There are many types of stores.
-  See [`nix help-stores`](@docroot@/command-ref/new-cli/nix3-help-stores.md) for a complete list.
+  There are many types of stores, see [Store Types](./store/types/index.md) for details.
+
+  [store]: #gloss-store
+
+- [local store]{#gloss-local-store}
 
-  From the perspective of the location where Nix is invoked, the Nix store can be  referred to _local_ or _remote_.
-  Only a [local store]{#gloss-local-store} exposes a location in the file system of the machine where Nix is invoked that allows access to store objects, typically `/nix/store`.
-  Local stores can be used for building [derivations](#gloss-derivation).
   See [Local Store](@docroot@/command-ref/new-cli/nix3-help-stores.md#local-store) for details.
 
-  [store]: #gloss-store
   [local store]: #gloss-local-store
 
```

Comment:
Thanks, but I would go further and delete all of this. There is no current `#gloss-local-store` and anything that wants to link to the local store can now just link to the local store page directly.

Reply:
Alright, done.